src/components/Channel/channelState.ts

```diff
@@ -178,6 +178,7 @@ export const channelReducer = <
       const { channel, message } = action;
       return {
         ...state,
+        quotedMessage: undefined,
```

Comment:
I'd like to deselect quoted message if a thread is opened, but that didn't work with this code

Reply:
also tried `setQuotedMessage` in `Channel.tsx` but no luck with that

Reply:
Hi Zita - we shouldn't need to reset this value within the reducer because the handle function to 'Reply' only sets that local state variable using [setQuotedMessage](https://github.com/GetStream/stream-chat-react/blob/develop/src/components/MessageActions/MessageActionsBox.tsx#L157). I was able to get everything to correctly update by adding `setQuotedMessage(undefined)` within the `openThread` method in `Channel.tsx`. This correctly updates the `ChannelStateContextValue`. Let me know if you're seeing differently or if I misunderstood your intent.

Reply:
Thanks, I moved the logic to the `Channel.tsx`, but that prompted another question (see my other comment)